src/zoid/button/component.jsx

```diff
@@ -245,14 +269,14 @@ export const getAuthButtonComponent = memoize(() : ZoidComponent<ButtonProps> =>
         }
     });
 
-    const ButtonWrapper = (props = {}) => {
+    const ButtonWrapper = (props) => {
         const instance = AuthButton(props);
-
-        instance.isEligible = () => {
-            const { fundingSource = FUNDING.PAYPAL } = props;
-            const fundingEligibility = getRefinedFundingEligibility();
-            return Boolean(fundingEligibility[fundingSource] && fundingEligibility[fundingSource].eligible);
-        };
+        // instance.isEligible = () => {
```

Comment:
Please remove anything you don't need, rather than commenting.

Reply:
done